tasks/Preprocess.cpp

```diff
@@ -26,38 +26,42 @@ Preprocess::~Preprocess()
 // documentation about them.
 bool Preprocess::configureHook()
 {
-    if (! PreprocessBase::configureHook())
+    if (!PreprocessBase::configureHook())
         return false;
-    if(_undistort.value())
+    if (_undistort.value())
         frame_helper.setCalibrationParameter(_calibration_parameters.value());
     return true;
 }
 
 bool Preprocess::startHook()
 {
-    if (! PreprocessBase::startHook())
+    if (!PreprocessBase::startHook())
         return false;
     return true;
 }
 
 void Preprocess::updateHook()
 {
     PreprocessBase::updateHook();
-    RTT::extras::ReadOnlyPointer<base::samples::frame::Frame> iframe;	
-    if(_iframe.read(iframe) != RTT::NewData)
+    RTT::extras::ReadOnlyPointer<base::samples::frame::Frame> iframe;
+    if (_iframe.read(iframe) != RTT::NewData)
         return;
 
-    Frame *frame_ptr = oframe.write_access();
-    frame_ptr->init(iframe->size.width*_scale_x-_offset_x,
-            iframe->size.height*_scale_y-_offset_y,
-            iframe->getDataDepth(),_format);
-    try
-    {
-        frame_helper.convert(*iframe,*frame_ptr,_offset_x.value(),
-                             _offset_y.value(),_resize_algorithm.value(),_undistort.value());
+    Frame* frame_ptr = oframe.write_access();
+    frame_ptr->init(iframe->size.width * _scale_x - _offset_x,
+        iframe->size.height * _scale_y - _offset_y,
+        iframe->getDataDepth(),
+        _format);
+    try {
+        frame_helper.convert(*iframe,
+            *frame_ptr,
+            _offset_x.value(),
+            _offset_y.value(),
+            _resize_algorithm.value(),
+            _undistort.value(),
+            _undistort_algorithm.value());
     }
-    catch(std::runtime_error e)
-    {
+    catch (std::runtime_error e) {
```

Comment:
Since you are refactoring anyways, please change this to catch by const-reference